src/Tools/BuildValidator/DemoLogger.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.Extensions.Logging;
+
+namespace BuildValidator
+{
+    internal sealed class DemoLogger : ILogger
```

Comment:
Can we add a TODO of either we should use the real logger or not log at all? I added this just for the Demo cause I couldn't figure out some issues with the default logger. Fine for this to go in now but think we need a TODO tracking cleaning this up.

Reply:
Fixed